packages/blaze-html-templates/.versions

```diff
@@ -17,7 +17,6 @@ htmljs@1.0.11
 id-map@1.0.8
 jquery@1.11.9
 meteor@1.6.0
-minifier-js@1.2.14
```

Comment:
I think you're probably right, but I just want to be sure the old file doesn't get used somehow.